dev/sparktestsupport/shellutils.py

```diff
@@ -55,7 +55,7 @@ def run_cmd(cmd, return_output=False):
         cmd = cmd.split()
     try:
         if return_output:
-            return subprocess_check_output(cmd).decode(sys.stdout.encoding)
+            return subprocess_check_output(cmd).decode(sys.getdefaultencoding())
```

Comment:
Change to `sys.getdefaultencoding()` because some env the `sys.stdout.encoding`, which break `bytestr.decode()` in python2